### PR TITLE
Ready? Set? Action! (Add ActionUtil tests)

### PR DESCRIPTION
**Pull request by Geweldig**
Adds tests for the ActionUtil class.

### CODE REVIEW

File: src/test/java/nl/tudelft/contextproject/util/webinterface/ActionUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * Test for the ActionUtil class.
+ */
+public class ActionUtilTest extends TestBase {
+	Game mockedGame;
```
Comment (Taeir) [inline src/test/java/nl/tudelft/contextproject/util/webinterface/ActionUtilTest.java]: Should be private or protected.

**Comment by Taeir** [comment]
Nice and simple. Apart from my one little comment, I think this should be merged.

File: src/test/java/nl/tudelft/contextproject/util/webinterface/ActionUtilTest.java (hunk 1 above)
Comment (Geweldig) [inline src/test/java/nl/tudelft/contextproject/util/webinterface/ActionUtilTest.java]: Whoops, completely agree and accidentally forgot.

**Comment by Taeir** [comment]
Should be merged.

**Comment by WytzeElhorst** [comment]
looks good to me

**Pull request merged**